cmd/profile/create.go

```diff
@@ -16,7 +16,7 @@ func createProfileCommand() *cobra.Command {
 	var arlonNs string
 	var argocdNs string
 	var desc string
-	var bundles string
+	var bundles = make([]string, 1)
```

Comment:
Is it necessary to make() the variable if it's going to be reset to nil by StringSliceVar()?

Reply:
will test and post the results.

Reply:
works

Reply:
works! the change is in